test/unexpectedMitm.js

```diff
@@ -1335,6 +1336,13 @@ describe('unexpectedMitm', function () {
             }, 'to yield response', 405);
         });
 
+        it('should preserve the fulfilment value', function () {
+            return expect('foo', 'with http recorded', 'to match', /^(f)o/)
+                .then(function (matches) {
+                    expect(matches, 'to satisfy', {0: 'fo', 1: 'f', index: 0});
+                });
+        });
```

Comment:
This is super pedantic, but to be consistent with other parts of mitm code/tests could we put .then() on the line above and unindent the expect on 1342?